references/obj_detection/train_pytorch.py

```diff
@@ -21,32 +22,57 @@
 
 from doctr.datasets import DocArtefacts
 from doctr.utils import DetectionMetric
+from doctr.transforms.functional.pytorch import rotate
 
 
-def convert_to_abs_coords(targets, img_shape):
+def convert_to_abs_coords(targets, img_shape, is_transform=False, **kwargs):
     height, width = img_shape[-2:]
-
     for idx in range(len(targets)):
         targets[idx]['boxes'][:, 0::2] = (targets[idx]['boxes'][:, 0::2] * width).round()
         targets[idx]['boxes'][:, 1::2] = (targets[idx]['boxes'][:, 1::2] * height).round()
-
-    targets = [{
-        "boxes": torch.from_numpy(t['boxes']).to(dtype=torch.float32),
-        "labels": torch.tensor(t['labels']).to(dtype=torch.long)}
-        for t in targets
-    ]
-
-    return targets
+    if is_transform:
```

Comment:
could you add some comments please? I don't get what `is_transform` is supposed to do :thinking:

Reply:
For sure :) 
`is_transform,` which is now changed to `use_aug` does the following:
It accepts a bool response, and if `use_aug = True`, augmentations will be employed and subsequently, images will be accepted as kwargs in  `convert_to_abs_coords(targets, img_shape, use_aug=False, **kwargs)`. If `use_aug= False`, we will not have to add images as kwargs. And the training will continue normally as it would without the augmentations.